Practice/case.py

```diff
@@ -4,12 +4,13 @@
 
 def get_input():
     """
-    Takes the input from the user"""
+    Takes the input from the user
+    """
     print('Please enter a word to be formatted:')
     return input()
 
 
-def test_format(word):
+def test_for_format_type(word):
```

Comment:
Think of a better name for this function. It's not really "testing" in the sense that testing is usually used. Testing implies comparison or if something works.

Maybe `find_case` or `determine_case`? You use the variable name `case` everywhere else, so parallel that.